CreateGame.py

```diff
@@ -0,0 +1,76 @@
+import logging, uuid, datetime, boto3, json
+from botocore.exceptions import ClientError
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+
+def handler(event, context):
+    logger.info('Handling event {} - context {}', event, context)
+
+    responseMessage = event["data"]["response"]["sms"]
+    errors = event["data"]["errors"]
+
+    # Initial game state
+    game = {
+        "gameState": "over"
+    }
+
+    dynamodb = boto3.resource('dynamodb', region_name='us-east-1')
+    hangmanGame = dynamodb.Table('HangmanGame')
+
+    # Get the game from the game table
+    response = hangmanGame.scan()
+    gameExists = len(response["Items"]) == 1
```

Comment:
We aren't going to store old games?

Reply:
I think we should. I'll rework CreateGame